src/plugins/chosen-selectpicker/plugin.js

```diff
@@ -23,7 +23,15 @@ QueryBuilder.define('chosen-selectpicker', function(options) {
     });
 
     this.on('afterCreateRuleOperators', function(e, rule) {
-        rule.$el.find(Selectors.rule_operator).removeClass('form-control').chosen(options);
+        var node = rule.$el.find(Selectors.rule_operator);
+        if (node.css('display') !== 'none') { // avoid creating useless chosen widget
```

Comment:
`self.getOperators(rule.filter).length > 1` would be a better test

Reply:
I guess you mean ``e.builder.getOperators(rule.filter).length > 1`` ?